README.md

```diff
@@ -230,6 +230,10 @@ ssl {
   // useful for self-signed certificates or for organizations using their own
   // internal certificate authority.
   ca_cert = "/path/to/ca"
+
+  // This is the path to a directory of PEM-encoded CA cert files. If both
+  // `ca_cert` and `ca_path` is specified, `ca_cert` is preferred.
+  ca_path = "path/to/certs/"
```

Comment:
Hmm. I see some of this might be leaking from Vault actually. I'm not 100% sure we should copy that UX. It feels weird to me to have two variables that effectively do the same thing - one is just a directory while the other is a path to a specific file.